eslzArm/managementGroupTemplates/policyAssignments/AUDIT-CostOptimizationPolicyAssignment.json

```diff
@@ -0,0 +1,80 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-08-01/managementGroupDeploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
```

Comment:
```suggestion
    "contentVersion": "1.0.0",
```

Reply:
This is incorrect @Springstone this content version is related to ARM and should be x.x.x.x as per https://learn.microsoft.com/en-us/azure/azure-resource-manager/templates/syntax